client/my-sites/email/email-management/home/email-plan-warnings.tsx

```diff
@@ -21,17 +22,18 @@ type EmailPlanWarningsProps = {
 
 const EmailPlanWarnings = ( { domain, emailAccount }: EmailPlanWarningsProps ) => {
 	const translate = useTranslate();
-	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const selectedSite = useSelector( getSelectedSite );
+	const selectedSiteSlug = selectedSite?.slug ?? '';
 
 	const warning = emailAccount?.warnings?.[ 0 ];
```

Comment:
To help distinguish the messaging shown when users cannot make a purchase from the messaging shown when users have need to finish setting up the email/mailboxes, should we rename `warning` to `requiredAction`?

![image](https://user-images.githubusercontent.com/104910361/193648351-51c6e0fa-2806-4e07-b3cf-cf8cffb69aa4.png)